packages/flutter_tools/lib/src/migrations/swift_package_manager_integration_migration.dart

```diff
@@ -557,69 +565,87 @@ $newContent
             .toList()
             .isNotEmpty;
     if (logErrorIfNotMigrated && !migrated) {
-      logger.printError('PBXNativeTarget was not migrated or was migrated incorrectly.');
+      logger.printError('Some PBXNativeTargets were not migrated or were migrated incorrectly.');
```

Comment:
Not sure if we should list the names of the affected targets? I think for this fix we can keep it without the names and improve on this later?